searcch_backend/models/model.py

```diff
@@ -26,7 +26,7 @@ class FileContent(db.Model):
 
     id = db.Column(db.Integer, primary_key=True, autoincrement=True)
     content = db.Column(db.LargeBinary(), nullable=False)
-    hash = db.Column(db.Binary(32), nullable=False)
+    hash = db.Column(db.LargeBinary(32), nullable=False)
```

Comment:
Ah, I see why you did this.  But it can't happen in this way, as described above, so let's pend it to a future migration if we really care.